circle.yml

```diff
@@ -1,16 +1,18 @@
 machine:
-  timezone: America/Los_Angeles
   php:
-    version: 5.5.0
+    version:
+      5.5.31
 
 dependencies:
   override:
     - mkdir -p build/logs
+    - echo "date.timezone = America/Los_Angeles" > /opt/circleci/php/$(phpenv global)/etc/conf.d/date.ini
     - composer update --prefer-stable --prefer-lowest --no-interaction
-    - sed -i 's/^;//' ~/.phpenv/versions/$(phpenv global)/etc/conf.d/xdebug.ini
+    - composer install-devtools
+    - phpenv versions
 
 test:
   override:
-    - phpunit
+    - devtools/vendor/bin/phpunit
```

Comment:
OK. So, I'll change it to PHP 7.1.0.

```
$ phpenv versions

  system
* 5.5.31 (set by /opt/circleci/.phpenv/version)
  5.5.32
  5.5.36
  5.6.17
  5.6.18
  5.6.22
  7.0.11
  7.0.4
  7.0.7
  7.1.0
```
via https://circleci.com/gh/KazuakiM/php-coveralls/170#action-12